gui/tsconfig.json

```diff
@@ -37,15 +37,12 @@
       "@SpCore/gists/*": ["app/gists/*"],
       /* We are playing a bit fast and loose with the distinction */
       "@SpPages/*": ["app/pages/HomePage/*"],
-      "@SpStanc/*": [
-        "app/Stanc/*",
-        "app/CompilationServerConnectionControl/*",
-        "app/compileStanProgram/*"
-      ],
+      "@SpStanc/*": ["app/Stanc/*", "app/CompilationServerConnectionControl/*"],
```

Comment:
I could debate whether or not CompilationServerConnectionControl should also be grouped with this context, but I'm fine either way in this PR

Reply:
This is also in main branch. Let's address separately.